src/plugins/workspace/public/components/workspace_form/workspace_collaborator_table.tsx

```diff
@@ -77,15 +77,22 @@ interface Props {
   handleSubmitPermissionSettings: (permissionSettings: WorkspacePermissionSetting[]) => void;
 }
 
+type PermissionSettingWithAccessLevelAndDisplayedType = PermissionSetting & {
+  accessLevel?: string;
```

Comment:
It seems we have `WorkspaceCollaboratorAccessLevel` for accessLevel.

Reply:
perhaps should be string? as we use PermissionSettingWithAccessLevelAndDisplayedType to identify items which retrieves its accessLevel by：
accessLevel: collaborator
          ? WORKSPACE_ACCESS_LEVEL_NAMES[collaborator.accessLevel]
          : undefined,
And  WORKSPACE_ACCESS_LEVEL_NAMES uses WorkspaceCollaboratorAccessLevel as the key and returns a string as the value.

Reply:
Oh, it should be string then as we are using the value of WORKSPACE_ACCESS_LEVEL_NAMES map.